controllers/slicegateway/slicegateway.go

```diff
@@ -621,6 +621,12 @@ func (r *SliceGwReconciler) SendConnectionContextToGwPod(ctx context.Context, sl
 		return ctrl.Result{RequeueAfter: 10 * time.Second}, nil, true
 	}
 
+	err = r.WorkerGWSidecarClient.UpdateSliceQosProfile(ctx, sidecarGrpcAddress, slice)
```

Comment:
To verify that this is working, please login to the gw pod and run these commands:
1. tc qdisc sh dev tun0
The output will contain the bw ceiling number, verify it matches the configured value in the slice's QoS section.

2. iptables -t mangle -L
You should see an entry in the POSTROUTING chain. The entry should have: DSCP, the udp nodeport and the dscp class configured in the qos profile.